csun-metalab/nomi
# NAM-164-1 -- This branch fixes the error in NAM-164, REPLACEMENT NAM-164

Description: 
This involves a few separate things

1) Remove Major from student profile
2) Clean up emails to be sanitized w/o "nr_"
3) Change default bio from "none" to "Pending biography from student."

Testing: 
1) check files
2) view routerlinks where nr_email was applied
3) check student profile for major removal and updated bio
